pkg/pillar/scripts/device-steps.sh

```diff
@@ -191,6 +173,36 @@ else
     echo "$(date -Ins -u) No separate $PERSISTDIR partition"
 fi
 
+DIRS="$CONFIGDIR $CONFIGDIR/DevicePortConfig"
+DIRS="$DIRS $TMPDIR $TMPDIR/DeviceNetworkConfig/ $TMPDIR/AssignableAdapters"
+DIRS="$DIRS $PERSISTDIR $PERSIST_RKT_DATA_DIR $PERSIST_RKT_CONF_LOCAL_DIR $PERSIST_RKT_CONF_LOCAL_AUTH_DIR"
```

Comment:
Does device-steps.sh depend on these new rkt directories? If not I suggest that they should be created by the relevant microservice which is actually using the directories. We need to make device-steps.sh simpler and with less dependencies to we can start the microservices more independently.

Reply:
Hey @eriknordmark -- just curious -- what's the typical policy we have for different microservices? Are they expected to simply go into /persist and create the folders they wish?

Reply:
Yes.. we can remove creation of these to Downloader.

Reply:
/persist/rkt directory is being used by two microservcies, downloader as well as domain manager. Also looks like rkt tool expects /persist/rkt (via softlink /var/lib/rkt) to be present right from boot time. Otherwise the results can be inconsistent.